docs/content/latest/integrations/metabase.md

```diff
@@ -105,6 +107,7 @@ Detailed steps on how to use Metabase are available in the [Metabase documentati
 - Filter data using WHERE clauses
 - Join data between tables
 - Perform data aggregation using GROUP BY
-- Use built-in functions such as SUM, MIN, MAX, etc.
+- Use built-in functions such as SUM, MIN, MAX, and so on
+
+Click **Ask a Question > Custom Query**. Choose the database you just set up, and enter the SQL queries noted in the [Retail Analytics](../../develop/realworld-apps/retail-analytics/) section.
```

Comment:
I wonder if the Retail Analytics section could be combined with this one?

Reply:
Please feel free to create an improvement ticket for that